src/main/java/seedu/address/logic/commands/UpCommand.java

```diff
@@ -0,0 +1,40 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.ItemModel;
+
+/**
+ * Command for scrolling up.
+ */
+public class UpCommand extends Command {
+
+    public static final String COMMAND_WORD = "up";
+    public static final String MESSAGE_SUCCESS = "Scrolling up...";
+    public static final String MESSAGE_USAGE = "Message Usage";
+
+    private String pane;
+
+    public UpCommand(String input) {
+        switch(input.trim()) {
+        case "L": case "l":
+            this.pane = "resultDisplay";
+            break;
+        case "R": case "r":
```

Comment:
I think that it might be better to split this into two lines so that it can improve readability.

Reply:
Done.